workspaces/diff-engine/src/queries/endpoint.rs

```diff
@@ -1,3 +1,4 @@
+use crate::events::SpecEvent::EndpointEvent;
```

Comment:
This doesn't seem to be used.

Reply:
- [ ] remove unused import that webstorm added